packages/manager/src/views/WebPropertyEnvPage/WebPropertyEnvPage.tsx

```diff
@@ -167,6 +184,12 @@ export const WebPropertyEnvPage = (): JSX.Element => {
         shortKey: popUp.deleteApiKey.data as string,
         propertyIdentifier
       });
+      const newDataForAPI = dataForAPI.filter((item) => item.shortKey !== popUp.deleteApiKey.data);
```

Comment:
U dont need to do this.

Just invalidate the list api call and it will be re-triggered